docs/how-to/h-enable-monitoring.md

```diff
@@ -0,0 +1,215 @@
+# Enable Monitoring
+
+The Charmed Spark solution comes with the [spark-metrics](https://github.com/banzaicloud/spark-metrics) exporter.
+This exporter is designed to push metrics to the [prometheus pushgateway](https://github.com/prometheus/pushgateway), that is integrated with the [Canonical Observability Stack](https://charmhub.io/topics/canonical-observability-stack). 
+
+
+To enable monitor with Charmed Spark just follow this guide.
+
+
+As a prerequisite, you need to have Juju 3 installed with a MicroK8s controller bootstrapped. This can be done following this [tutorial](https://charmhub.io/topics/canonical-observability-stack/tutorials/install-microk8s).
+
+
+As a first step, start by deploying cos-lite bundle in a Kubernetes environment with Juju.
```

Comment:
I would add the `prometheus-pushgateway` here as well. I don't think it needs to be deployed after the cos bundle